rs/rust_canisters/load_simulator/src/main.rs

```diff
@@ -0,0 +1,81 @@
+use ic_cdk::api::stable;
+use std::cell::RefCell;
+use std::time::Duration;
+
+thread_local! {
+    static COUNTER: RefCell<i32> = const { RefCell::new(0) };
+}
+
+async fn timer_handler() {
+    COUNTER.with(|counter| {
+        let mut counter = counter.borrow_mut();
+        *counter += 1;
+
+        // Write to stable memory every 50 timer calls.
+        if *counter % 50 == 0 {
+            if stable::stable_size() < 5 {
+                let _ = stable::stable_grow(1);
+            }
+            if stable::stable_size() > 0 {
+                stable::stable_write(0, &[1, 2, 3]);
+            }
+        }
+    });
+}
+
+#[ic_cdk::update]
+fn no_op_update() {
+    // Do nothing.
+}
+
+#[ic_cdk::query]
+fn no_op_query() {
+    // Do nothing.
+}
+
+#[ic_cdk::init]
+fn set_up_timer() {
+    // Set up a canister timer to call a function every N seconds.
+    ic_cdk_timers::set_timer_interval(Duration::from_secs(1), || ic_cdk::spawn(timer_handler()));
```

Comment:
Have you considered using a heartbeat vs. this timer? Maybe the difference doesn't matter for your use case, in which case please ignore this comment.

Reply:
It's simply to mimic a load on the subnet. Unlike heartbeats, timers also generate self-calls and replies, which better represents the load we see on some subnets.